webapp/src/components/common/SplitView/index.tsx

```diff
@@ -87,7 +87,11 @@ function SplitView({
       direction={direction}
       sizes={activeSizes ?? defaultSizes}
       minSize={minSize}
-      onDragEnd={setActiveSizes} // Update sizes on drag end.
+      onDragEnd={(sizes) => {
+        // Round to whole numbers to avoid excessive precision
+        const roundedSizes = sizes.map((size) => Math.round(size));
```

Comment:
Better to round in the useEffect: `storage.setItem(localStorageKey, activeSizes.map(Math.round));`.
No ne need to change the current value.